### PR TITLE
enable mypy check in nnoir_onnx

### PR DESCRIPTION
**Pull request by eguchi1904**
Add type annotation to nnoir_onnx.

Throughout the nnoir_onnx source code, there are numerous mismatches between `bytes` and `str`. As a result, `type: ignore` is frequently used.

### CODE REVIEW

File: nnoir-onnx/nnoir_onnx/onnx.py [hunk 1]
```diff
@@ -446,22 +454,9 @@ def is_used(name):
         )
 
 
-def to_dummy_input(x):
```
Comment (eguchi1904) [inline nnoir-onnx/nnoir_onnx/onnx.py:449]: remove because this function is unused

File: .circleci/config.yml [hunk 2]
```diff
@@ -84,11 +84,6 @@ jobs:
           dir: nnoir
       - lint:
           dir: nnoir
-      - run:
-          name: manual mypy check # workaround https://github.com/pfnet/pysen/issues/32
-          working_directory: nnoir
-          command: |
-            poetry run mypy . --strict --implicit-reexport --ignore-missing-imports
```
Comment (eguchi1904) [inline .circleci/config.yml:91]: use mypy via Pysen

File: nnoir/nnoir/py.typed
Comment (eguchi1904) [inline nnoir/nnoir/py.typed:1]: https://peps.python.org/pep-0561/#packaging-type-information

**Review by ishiy1993 — APPROVED** [review]
LGTM

**Comment by eguchi1904** [comment]
:pray:

**Pull request merged**